plc4j/drivers/mock/src/main/java/org/apache/plc4x/java/mock/connection/MockConnection.java

```diff
@@ -221,4 +224,14 @@ public PlcAuthentication getAuthentication() {
         return authentication;
     }
 
+    @Override
+    public Optional<PlcTag> parseTagAddress(String tagAddress) {
+        PlcTag plcTag;
+        try{
+            plcTag = mockTagHandler.parseTag(tagAddress);
+        }catch (Exception e){
```

Comment:
the Exception should be at least logged at a debug level or if it never matters the variable should be renamed to `ignore`